runtime/onert/frontend/base_loader/include/base_loader.h

```diff
@@ -354,18 +358,43 @@ ir::OperandIndex BaseLoader<LoaderDomain>::loadOperand(const Tensor *tensor, ir:
       ptrdiff_t aligned_offset_start = (unaligned_offset_start / _pagesize) * _pagesize;
       size_t mmap_size = offset_end - aligned_offset_start;
 
+      uint32_t buf_idx = tensor->buffer();
+      auto buffer_found = _buf_to_data.find(buf_idx);
+
       if (_use_mmaped_data)
       {
-        data_obj = std::make_unique<ir::MMapedData>(_fd, aligned_offset_start, mmap_size,
-                                                    unaligned_offset_start, data_size);
+        if (buffer_found != _buf_to_data.end())
```

Comment:
Optional) How about moving this if statement outside of `if (_use_mmaped_data)` to remove code duplication?

Reply:
Thanks. Fixed.